### ISSUE TITLE
Event format date question

### ISSUE THREAD

**Issue by Alex-Ascet** [ThemeFuse/Unyson#1096]
Hi there,

1- Is it possible to change the way to display the date to this way d/mY? In which file? I have tried to change the class-fw-extension-event.php without any results.

2- Is it possible to separate the date and the hour data of the event? Currently all data (date and hours) come with ['event_date_range']['from'] and ['event_date_range']['to']

Thank you very much for your time and this framework. Definitely love it!

Regards

**Comment by Alex-Ascet** [comment]
Any advice? Thank you.

**Comment by Alex-Ascet** [comment]
Any update?

**Comment by Alex-Ascet** [comment]
@llemurya 
Please, I definitely need an update on this feed. How can I change to d/m/Y on the event section (backend and frontend), Thank you for you help.

**Comment by llemurya** [comment]
@Alex-Ascet in the new version of the events I added an hook, you can change the datetime-pickers
```php
add_filter('fw_option_type_event_datetime_pickers', function($data){
	$data['from']['format'] = 'd/mY';
	$data['to']['format'] = 'd/mY';

	return $data;
});
```

**Comment by Alex-Ascet** [comment]
@llemurya Thank you for your support, I really appreciate this new feature and help from you.

The step filter I have found here:
https://github.com/ThemeFuse/Unyson-Events-Extension/issues/15 
works perfectly but not the date format one. Here my code:

```
add_filter('fw_option_type_event_datetime_pickers', function($data){
    $data['from']['format'] = 'd/m/Y';
    $data['to']['format'] = 'd/m/Y';

    return $data;
});
add_filter('fw_option_type_event_datetime_pickers', function($data){
    $data['from']['step'] = 15;
    $data['to']['step'] = 15;

    return $data;
});
```
I have tried d/m/Y DD/MM/YYY and dmy without results.

This error appears:
![screen shot 2016-01-05 at 12 12 42 pm](https://cloud.githubusercontent.com/assets/12162252/12105107/ad2b6770-b3a5-11e5-9bb7-b828d0c9c894.png)

Thank you for your time.
